api/MediaKeySystemConfiguration.json

```
@@ -161,10 +161,10 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "44"
```

```suggestion
              "version_added": "43"
```

https://hg.mozilla.org/releases/mozilla-release/file/FIREFOX_43_0_RELEASE/dom/webidl/MediaKeySystemAccess.webidl#l20